src/app/inventory/Stores.tsx

```diff
@@ -80,6 +85,15 @@ function Stores(this: void, { stores, buckets, isPhonePortrait }: Props) {
     }
   };
 
+  const handleCategoryChange = (category: string) => {
+    if (category === selectedCategoryId) {
+      // If user selects the category they are already on, scroll to top
+      scrollToPosition({ top: 0 });
```

Comment:
You might instead consider a `useEffect` triggered on `selectedCategoryId` changing?